README.md

```diff
@@ -24,16 +25,25 @@ Share text using a native sharing dialog.
 **PARAMETERS**
 * ```text``` (string) - The text to share
 
-### share.image(bytes, [text])
-Share an image (with optional text) using a native sharing dialog. The image format must be supported by ```BitmapFactory.decodeByteArray()``` on Android and ```UIImage.initWithData``` on iOS/OSX. On Android the image will first be stored locally as a file and then shared using a FileProvider. In order for this to work you need to follow the additional setup steps for Android mentioned above.
+### share.image(bytes, [text], [file_name])
+Share an image (with optional text) using a native sharing dialog. The image format must be supported by ```BitmapFactory.decodeByteArray()``` on Android, ```UIImage.initWithData``` on iOS/OSX and [DataURI](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs) on HTML5. On Android the image will first be stored locally as a file and then shared using a FileProvider. In order for this to work you need to follow the additional setup steps for Android mentioned above.
 
 **PARAMETERS**
 * ``` bytes``` (string) - The image bytes to share
 * ```text``` (string) - Optional text to share
+* ```file_name``` (string) - Optional Only for `HTML5` image name. Default `"file.png"`
 
-### share.file(path, [text])
+### share.file(path, [text], [data])
 Share a file (with optional text) using a native sharing dialog. On Android the file will first be copied to a predefined location and then shared using a FileProvider. In order for this to work you need to follow the additional setup steps for Android mentioned above. Files are shared with their original filename and extension. This will allow iOS to offer different kinds of applications depending on the shared content.
 
 **PARAMETERS**
-* ``` path``` (string) - Full path to the file to share
-* ```text``` (string) - Optional text to share
+* ``` path``` (string) - Full path to the file to share. For `HTML5`: specify the name of the file. Default `file.txt`.
+* ```text``` (string) - Optional text to share. For `HTML5`: specify the data that should be in the file.
+
+* ```data``` (table) - Optional Only for `HTML5`.
```

Comment:
It's not data is it? More like config values or options. We usually call this table `options` (see http.request for instance)